src/inference/src/dev/preprocessing/mean_image.cpp

```diff
@@ -32,7 +31,7 @@ ngraph::pass::AddMeanSubtract::AddMeanSubtract(const MeanMap& inputInfoMap) {
                      " must have f32 type");
 
         auto copy_param = param->clone_with_new_inputs({});
-        auto sub = std::make_shared<ngraph::opset3::Subtract>(copy_param, mean_const);
+        auto sub = std::make_shared<ov::op::v1::Subtract>(copy_param, mean_const);
```

Comment:
Do we have tests  for this functionality? 

Can we also deprecated mean image in IE API?

Reply:
Yes we have tests for this functionality.
I think we can do it together with IE API